force-app/main/userCommunity/lwc/hot_createRequestForm/hot_createRequestForm.js

```diff
@@ -407,13 +401,27 @@ export default class RecordFormCreateExample extends NavigationMixin(LightningEl
         return recurringTypeValid && recurringDaysValid && recurringEndDateValid;
     }
 
+    checkboxValue = false;
+    handleCheckboxValue(event) {
+        this.checkboxValue = event.detail;
+        this.template.querySelector('c-checkbox').validationHandler(''); // Clear validation when clicking checkbox. Only validate on Submit.
+    }
+
     handleValidation() {
+        let checkboxValid = true;
+        if (this.fileData.length > 0) {
+            if (!this.checkboxValue) {
+                this.template.querySelector('c-checkbox').validationHandler('Sjekkboks må være fylt ut');
```

Comment:
Kan kanskje ha en litt bedre feilmelding?
"Du må gi samtykke til at formidler og tolk får se vedlegg for å legge til vedlegg?"
Noe sånt? Er jo flere løsninger som har noe slikt. Akseptere betingelser og sånn. Hva er det de skriver om man ikke huker av?